src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Utils/ImporterUtils.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Utils
+{
+    public static class ImporterUtils
+    {
+        public static readonly List<GeographicLevel> IgnoredGeographicLevels = new List<GeographicLevel>
+        {
+            GeographicLevel.Institution,
+            GeographicLevel.Provider,
+            GeographicLevel.School,
+            GeographicLevel.PlanningArea
+        };
+
+        private static readonly List<GeographicLevel> AllowedSoloGeographicLevels = new List<GeographicLevel>
+        {
+            GeographicLevel.Provider,
+            GeographicLevel.School,
+        };
+
+        /// <summary>
+        /// If a subject contains a single geographic level that is contained in AllowedSoloGeographicLevels, then
+        /// we import every CSV row of that subject's data file.
+        /// </summary>
+        /// <remarks>
+        /// If a subject contains more than one geographic level or isn't included in AllowedSoloGeographicLevels,
+        /// we then defer to AllowRowImport to determine if a particular row should be imported.
+        /// </remarks>
+        public static bool HasSoloAllowedGeographicLevel(HashSet<GeographicLevel> subjectGeographicLevels)
+        {
+            if (subjectGeographicLevels.Count == 1)
```

Comment:
I think this whole method can be simplified to `return subjectGeographicLevels.Count == 1 && AllowedSoloGeographicLevels.Contains(subjectGeographicLevels[0]))`